src/store.js

```diff
@@ -156,6 +156,18 @@ export class Store {
         }
       }
       return res
+    }, e => {
+      try {
+        this._actionSubscribers
+          .filter(sub => sub.catch)
+          .forEach(sub => sub.catch(action, this.state, e))
+      } catch (_e) {
+        if (process.env.NODE_ENV !== 'production') {
+          console.warn(`[vuex] error in after action catch subscribers: `)
+          console.error(_e)
+        }
+      }
+      throw e
```

Comment:
Let's not throw here since this is technically a breaking change.

Reply:
@kiaking -  I was not aware this was a breaking change, as existing tests pass with the throw. Without throwing here, the catch handler on the original dispatch call does not get called, which actually breaks an existing test in store.spec.js  'detect action Promise errors'. Am I missing something here? Let me know how to proceed.

Reply:
@kiaking - Understand the dist folder, but please advise on the comment I added after yours regarding the throw.